jujugui/static/gui/src/app/components/deployment/deployment.js

```diff
@@ -31,6 +31,39 @@ YUI.add('deployment-component', function() {
       getUnplacedUnitCount: React.PropTypes.func.isRequired,
     },
 
+    componentDidMount: function() {
```

Comment:
I'm curious as to why you're navigating to start after mount and in render - doesn't this fire changeState then on the first render after it's already rendered the appropriate component.